recipes/cwl-utils/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "cwl-utils" %}
+{% set version = "0.18" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cwl-utils-{{ version }}.tar.gz
+  sha256: 43e57318db207310e46726df1081c55ceddebab888e09484d1175f4c9a103384
+
+build:
+  # Copied from schema-salad
+  skip: true  # [py2k or win]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - cwl-upgrader >=1.2
+    - packaging
+    - python
+    - requests
+    - schema-salad >=8.2,<9
+
+test:
+  imports:
+    - cwl_utils
+  commands:
+    - pip check
+    - cwl-cite-extract --help
+    - cwl-docker-extract --help
+    - cwl-expression-refactor --help
+    # TODO: cwl-utils is missing cwlformat from its dependencies
+    # - cwl-graph-split --help
```

Comment:
Skipping this one for now as Conda failed to run it. Reproduced locally with `pip`. See https://github.com/common-workflow-language/cwl-utils/pull/161